docs/getting_started.md

```diff
@@ -54,20 +61,21 @@ Once you have the dependencies satisfied run `pip install healthcareai` or `sudo
 
 ### Verify Installation
 
-To verify that *healthcareai* installed correctly, open a terminal and run `python`. This opens an interactive python console (also known as a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)). Then enter this command: `from healthcareai import SupervisedModelTrainer` and hit enter. If no error is thrown, you are ready to rock.
+To verify that *healthcareai* installed correctly:
+1. Open a terminal and run `python` or `ipython`. Either of these opens an interactive python console (also known as a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)).
+2. Then enter this command: `from healthcareai import SupervisedModelTrainer` and hit enter. If no error is thrown, you are ready to rock.
```

Comment:
in the getting started doc here from the master branch: https://github.com/HealthCatalyst/healthcareai-py/blob/master/docs/getting_started.md 

it says to verify installation by importing "develop_supervised_model."  Don't know if this helps or adds to confusion